File: qc-core/src/main/java/com/eager/questioncloud/core/domain/post/infrastructure/PostCommentJpaRepository.java
package com.eager.questioncloud.core.domain.post.infrastructure;

import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PostCommentJpaRepository extends JpaRepository<PostCommentEntity, Long> {
    Optional<PostCommentEntity> findByIdAndWriterId(Long id, Long userId);
}

File: qc-core/src/main/java/com/eager/questioncloud/core/domain/post/infrastructure/repository/PostCommentJpaRepository.kt
package com.eager.questioncloud.core.domain.post.infrastructure.repository;

import com.eager.questioncloud.core.domain.post.infrastructure.entity.PostCommentEntity;
import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PostCommentJpaRepository extends JpaRepository<PostCommentEntity, Long> {
    Optional<PostCommentEntity> findByIdAndWriterId(Long id, Long userId);
}
